src/aks-preview/HISTORY.rst

```diff
@@ -11,9 +11,16 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
-* Vendor new SDK and bump API version to 2023-11-02-preview.
 * Implicitly enable istio when ingress or egress gateway is enabled for Azure Service Mesh.
```

Comment:
shouldn't be a blocker to merge the PR, but why this line is left in the Pending section?